docs/transparency.md

```diff
@@ -0,0 +1,220 @@
+  then collates counter-signatures for a given checkpoint from one or more
+  ArmoredWitness devices, and serves them via a public API.
+
+The benefit of this system comes though removing trust from log operators to
+behave honestly, and placing some of that trust in the witnesses. Splitting the
+trust across multiple parties in this way means that a larger number of parties
+must collude to hide malfeasance, and as other witness implementations/networks
+start to appear, the number of parties required to collude increases
+correspondingly.
+
+However, we can minimise the amount of trust required to be placed in the
+ArmoredWitness by having it be as transparent as possible too.
+
+## Minimising trust
+
+In order to minimise the level of trust required to be placed in the ArmoredWitness,
+we have designed & implemented it such that:
+
+* All firmware is opensource, written in [TamaGo](https://github.com/usbarmory/tamago),
+  and is build-reproducible by anyone.
+* All firmware is logged to a Firmware Transparency log at build and release time.
+* The [`provision`](cmd/provision/) tool will only use firmware artefacts discovered
+  in the FT log in order to program devices.
+* The on-device self-update process requires that updated firmware is hosted in the 
+  FT log.
+* The boot "chain of trust" requires valid "off-line FT proof bundles" to be present
+  alongside the firmware at boot time:
+  * the bootloader verifies signatures and FT proofs for the secure monitor ("OS"),
+    and only launches it if they succeed
+  * the secure monitor ("OS") verifies signatures and FT proofs for the witness
+    applet, and only launches it if they succeed.
+* The [`verify`](cmd/verify) tool can be used by _custodians_ to inspect the device,
+  extract the firmware components from it, and verify that they are present in the FT log.
+* The [`verify_build`](cmd/verify_build) command continuously monitors the contents of
+  the FT log, and tests that every logged firmware is indeed reproducibly built.
+
+### Firmware Transparency
+
+Every piece of non-ROM firmware for the ArmoredWitness is automatically added to our
+FirmwareTransparency logs during the build & release process (e.g.
+[applet GCB config](https://github.com/transparency-dev/armored-witness/blob/843b7cf1f703698de40cf82aa7d0cc38fa76859d/deployment/build_and_release/modules/release/main.tf#L245-L465)
+). In conjunction with the controls in the firmware self-update, provision, and user
+verification tooling mentioned above, this helps to ensure that we, or anybode else,
+would find it very difficult to covertly target all, or a subset of the devices,
+with malicious firmware.
+
+Rather than writing compiled blobs directly into the log, we store a _manifest_, which
+commits to all necessary inputs to the build, and the corresponding firmware output.
+
+Here is a [real manifest from the CI log](https://storage.googleapis.com/armored-witness-firmware-log-ci-2/seq/00/00/00/00/2f):
+
+```json
```

Comment:
It's not technically json because of the note signature, which is probably why the signature is rendered in red (because it assumes its a diff?)

Reply:
Sad, but true.

I think red signature is probably because it's very not json, with a weird unquoted character in an unexpected place.

Anyway, dropped the tag for a more monochromatic life.